04/app.js

```diff
@@ -0,0 +1,20 @@
+const boxes = document.querySelectorAll('div');
+
+boxes.forEach(box => {
+    box.addEventListener('click', (e) => {
+        e.stopPropagation();
```

Comment:
Spróbuj wykonać to bez zatrzymywania propagacji.
Poniższe rozwiązanie jest dość zawiłe przez niepotrzebne warunki.